docs/guide/repository.md

```diff
@@ -1549,6 +1549,124 @@ class AcmeFooBundle extends Bundle
 
 Policies used internally in repository services are defined in  `EzPublishCoreBundle/Resources/config/policies.yml` .
 
+### Custom Limitation UI
+
+To provide support for editing custom policies in Platform UI you need to implement [`EzSystems\RepositoryForms\Limitation\LimitationFormMapperInterface`](https://github.com/ezsystems/repository-forms/blob/master/lib/Limitation/LimitationFormMapperInterface.php):
+
+```php
+<?php
+
+/**
+ * Interface for LimitationType form mappers.
+ *
+ * It maps a LimitationType's supported values to editing form.
+ */
+interface LimitationFormMapperInterface
+{
+    /**
+     * "Maps" Limitation form to current LimitationType, in order to display one or several fields
```

Comment:
Why use quotation marks for "Maps" when above it is stated that it maps without quotation?